.pre-commit-config.yaml

```diff
@@ -57,8 +57,3 @@ repos:
     hooks:
       - id: mypy
         additional_dependencies: []
-  - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.10
-    hooks:
-      - id: bandit
-        args: [-x, tests]
```

Comment:
Kinda silly to include bandit since all these devices include insecure encryption